@xen-orchestra/web/src/remote-resources/use-xo-routes.ts

```diff
@@ -0,0 +1,14 @@
+import type { XoRoutes } from '@/types/xo/xo-routes.type.ts'
+import { defineRemoteResource } from '@core/packages/remote-resource/define-remote-resource.ts'
+
+export const useXoRoutes = defineRemoteResource({
+  url: '/rest/v0/gui-routes',
+  pollingIntervalMs: 0,
+  initialData: () => ({}) as XoRoutes,
```

Comment:
`XoRoutes` is typed with 2 mandatory properties.

I suggest this instead:

```suggestion
  initialData: () => undefined as XoRoutes | undefined,
```